sdk/keyvault/keyvault-admin/review/keyvault-admin.api.md

```diff
@@ -0,0 +1,102 @@
+## API Report File for "@azure/keyvault-admin"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import * as coreHttp from '@azure/core-http';
+import { PagedAsyncIterableIterator } from '@azure/core-paging';
+import { TokenCredential } from '@azure/core-http';
+
+// @public (undocumented)
+export class AccessControlClient {
+    constructor(vaultUrl: string, credential: TokenCredential, pipelineOptions?: AccessControlClientOptions);
+    createRoleAssignment(scope: RoleAssignmentScope, name: string, options?: CreateRoleAssignmentOptions): Promise<KeyVaultRoleAssignment>;
```

Comment:
In .NET, we're using `roleScope` for the parameter name.

Reply:
Addressed here: https://github.com/Azure/azure-sdk-for-js/pull/10815/commits/26e95837b87e1b804e309fe45421ee01ead5d165